io/net/bonding.py

```diff
@@ -51,6 +52,15 @@ def setUp(self):
         '''
         To check and install dependencies for the test
         '''
+        interfaces = netifaces.interfaces()
```

Comment:
@smruti77 This needs to be handled a little differently for bonding, since the IP configuration for bond interface is done as part of the test itself.